.dvcignore

```diff
@@ -0,0 +1,5 @@
+# Add patterns of files dvc should ignore, which could improve
+# the performance. Learn more at
+# https://dvc.org/doc/user-guide/dvcignore
+utils/juliaup
```

Comment:
```suggestion
```

This hasn't been used for a while. If you still have stuff there, you can delete it.